app/assets/javascripts/app/app.js

```diff
@@ -25,6 +26,10 @@ var App = window.App = Skull.View.extend({
     }
   },
 
+  toggleEdit: function(event){
+    $("#edit-menu-items").slideToggle( "slow" );
```

Comment:
Could we implement a CSS solution here? Do the transition on CSS, only changing the class, just like on the toggleNav function.